scripts/superdesk/editor2/editor.js

```diff
@@ -734,12 +734,14 @@ angular.module('superdesk.editor2', [
     }])
     .directive('sdTextEditorBlockText', ['editor', 'spellcheck', '$timeout', 'superdesk', '$q', 'gettextCatalog', 'config',
     function (editor, spellcheck, $timeout, superdesk, $q, gettextCatalog, config) {
+        var TOP_OFFSET = 134; // header height
```

Comment:
Is there any way we can retrieve the height programatically? I'm thinking if we change the CSS or skin in the future, this could cause issues. If that's too difficult or doesn't work, you can leave it as is.

P.S. If you prefer, we can now use `const` instead of `var` for constants.